packages/crew_profile_base.rb

```diff
@@ -0,0 +1,40 @@
+require 'package'
+
+class Crew_profile_base < Package
+  description 'Crew-profile-base sets up Chromebrew\'s environment capabilities.'
+  homepage 'https://github.com/chromebrew/crew-profile-base'
+  version '0.0.1-pre1'
+  license 'GPL-3+'
+  compatibility 'all'
+  source_url 'https://github.com/chromebrew/crew-profile-base/releases/download/0.0.1-pre1/crew-profile-base-0.0.1-pre1.tar.xz'
+  source_sha256 '22c41f65b2d6c3b858242216703230354f2697b4d6f410826f4a633da155dd98'
+
+  binary_url ({
+
+  })
+  binary_sha256 ({
+
+  })
+
+  def self.preinstall
+    @rcfile = <<~EOF
```

Comment:
The EOF for docfiles is supposed to be descriptive in ruby. so maybe `RCFILEEOF`

Reply:
I know it *can* be descriptive in ruby, just like it can in bash, but because the `@variable` is already descriptive, would it be okay if I left it for readability and simplicity purposes?